src/index/generateTrees.ts

```diff
@@ -17,7 +19,12 @@ export function generateTrees(restructureMap: { [key: string]: string[] }) {
       const { graph, files, useForwardSlash, parentFolder } = buildGraph(
         filePaths
       );
-      const tree = toFractalTree(graph, findEntryPoints(graph));
+
+      let tree = toFractalTree(graph, findEntryPoints(graph));
+
+      if (renameOnlyChild) {
```

Comment:
https://github.com/benawad/destiny/pull/89#discussion_r386085311
```suggestion
      if (avoidSingleFile) {
```